app/components/candidate_interface/course_choices_review_component.html.erb

```diff
@@ -1,5 +1,51 @@
+
+  <% if course_choice.course_not_available? %>
+    <div class='review-warning--primary-message'>
+      <p>You cannot apply to '<%= course_choice.course.name_and_code %>' because it is not running.</p>
+    </div>
+    <p class='govuk-body'><%= course_choice.provider.name %> have decided not to run this course.</p>
+    <p class='govuk-body'>You can:</p>
+    <ul class='govuk-list govuk-list--bullet'>
+      <li><%= link_to 'delete this choice', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %></li>
+      <li><%= link_to 'change to another course', course_change_path(course_choice), class: 'govuk-link', data: { action: :delete } %></li>
+      <li>
+        <%= link_to 'contact the training provider', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %>
+        to discuss alternatives
+      </li>
+    </ul>
+  <% elsif course_choice.course_full? %>
+    <div class='review-warning--primary-message'>
+      <p>You cannot apply to '<%= course_choice.course.name_and_code %>' because it has no vacancies.</p>
+    </div>
+    <p class='govuk-body'>You can:</p>
+    <ul class='govuk-list govuk-list--bullet'>
+      <li><%= link_to 'delete this choice', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %></li>
+      <li><%= link_to 'change to another course', course_change_path(course_choice), class: 'govuk-link', data: { action: :delete } %></li>
+      <li>
+        <%= link_to 'contact the training provider', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %>
+        to see if the course will re-open or discuss alternatives
+      </li>
+    </ul>
+  <% elsif course_choice.chosen_site_full? %>
+    <div class='review-warning--primary-message'>
+      <p class='govuk-body govuk-!-font-weight-bold'>Your chosen site for '<%= course_choice.course.name_and_code %>' has no vacancies.</p>
+    </div>
+
+    <p class='govuk-body'>You can:</p>
+    <ul class='govuk-list govuk-list--bullet'>
+      <li><%= link_to 'pick a new location', site_change_path(course_choice), class: 'govuk-link', data: { action: :delete } %></li>
+      <li><%= link_to 'delete this choice', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %></li>
+      <li><%= link_to 'change to another course', course_change_path(course_choice), class: 'govuk-link', data: { action: :delete } %></li>
+      <li>
+        <%= link_to 'contact the training provider', candidate_interface_confirm_destroy_course_choice_path(course_choice.id), class: 'govuk-link', data: { action: :delete } %>
+        to see if the course will re-open or discuss alternatives
+      </li>
+    </ul>
+  <% end %>
```

Comment:
All the links on this page can be `govuk_link_to`, saves you the `class: 'govuk-link'` 👍